cdap-app-fabric/src/main/java/io/cdap/cdap/app/runtime/AbstractProgramRuntimeService.java

```diff
@@ -138,7 +138,7 @@ public final RuntimeInfo run(ProgramDescriptor programDescriptor, ProgramOptions
       } catch (Exception e) {
         controller.failed(e);
         programStateWriter.error(programRunId, e);
-        LOG.error("Exception while trying to run program run {}", programRunId, e);
+        LOG.error("Exception while trying to run program with runID {}", programRunId, e);
```

Comment:
Why do you need this change? program run is a proper term

Reply:
Ok, I misinterpreted term "program run", so I changed it when I came across it. Will revert it back.